library/Zend/Paginator/PaginatorIterator.php

```diff
@@ -0,0 +1,113 @@
+<?php
+
+namespace Zend\Paginator;
+
+use OuterIterator;
+
+class PaginatorIterator implements OuterIterator
```

Comment:
@keradus 
Please report this at the [issue tracker for documentation](https://github.com/zendframework/zf2-documentation).